### PR TITLE
FND-312 - The Occurrences ontology includes a property, hasDescription, that is largely redundant with isDescribedBy and the concepts of occurrence and occurrence kind need clarification

### PR DESCRIPTION
**Pull request by ElisaKendall**
eliminated the hasDescription property from occurrences; cleaned up several definitions to make them ISO 704 compliant, eliminate circularity, and, in the case of occurrence kind, augment its semantics to provide clarity with respect to how it should be used

Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

Eliminated the hasDescription property from occurrences; cleaned up several definitions to make them ISO 704 compliant, eliminate circularity, and, in the case of occurrence kind, augment its semantics to provide clarity with respect to how it should be used

Fixes: #1149 / FND-312

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](../CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](../ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

**Review by dallemang — APPROVED** [review]
good

File: FND/DatesAndTimes/Occurrences.rdf [hunk 1]
```diff
@@ -177,24 +184,25 @@ They are modularized this way to minimize the ontological committments that are
 				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label>occurrence based date</rdfs:label>
-		<skos:definition>An OccurrenceBaseDate is a CalculatedDate that is defined with respect to the Occurrence of some OccurrenceKind. The &apos;hasDateValue&apos; property of an OccurrenceBasedDate is not set until the Occurrence happens.  The &apos;triggeredBy&apos; property relates an OccurrenceBasedDate to the OccurrenceKind that gives the meaning of the OccurrenceBasedDate.</skos:definition>
+		<rdfs:label>occurrence-based date</rdfs:label>
+		<skos:definition>calculated date that is defined with respect to the occurrence of some occurrence kind</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>The &apos;hasDateValue&apos; property of an OccurrenceBasedDate is not set until the Occurrence happens.  The &apos;triggeredBy&apos; property relates an OccurrenceBasedDate to the OccurrenceKind that gives the meaning of the OccurrenceBasedDate.</fibo-fnd-utl-av:explanatoryNote>
```
Comment (rivettp) [inline FND/DatesAndTimes/Occurrences.rdf:189]: I could really use an example and more around intended usage. Let's take a scheduled dividend payment date, which is set per a fixed schedule or declared a month or so in advance by the Board.  Would that be suddenly be additionally classified by OccurrenceBasedDate only once the payment happens? 
When is the Occurrence (e.g. a payment) deemed to have "happened"? This is important for the definition of this class. There are various options, taking as an example old fashioned means of payment: when the payment instruction is given, when the check is posted, wehn it arrives, when it is deposited, when the money leaves the sender's account, when it arrives in the receiver's account or when it has cleared in the receiver's account? Where does FIBO have the answer?

File: FND/DatesAndTimes/Occurrences.rdf [hunk 2]
```diff
@@ -177,24 +184,25 @@ They are modularized this way to minimize the ontological committments that are
...
+				<owl:onProperty rdf:resource="&lcc-cr;classifies"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-dt-oc;Occurrence"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>occurrence kind</rdfs:label>
-		<skos:definition>instances of OccurrenceKind are types of events, each having a description</skos:definition>
-		<skos:editorialNote>In order for other ontologies to accept FinancialDates without committing to the particular notions of &apos;Occurrence&apos; and &apos;OccurrenceKind&apos; that is modeled here, all aspects of Occurrences are captured in this ontolog</skos:editorialNote>
-		<skos:example>Loan origination</skos:example>
-		<skos:example>Trade settlement</skos:example>
+		<skos:definition>classifier that specifies the general nature of an occurrance (event)</skos:definition>
```
Comment (rivettp) [inline FND/DatesAndTimes/Occurrences.rdf:201]: typo - occurrence has no "a". 
Also I think "the general nature" could be more specific.Examples would be useful.

File: FND/DatesAndTimes/Occurrences.rdf [hunk 3]
```diff
@@ -257,19 +257,20 @@ They are modularized this way to minimize the ontological committments that are
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-dt-oc;isInputTo">
 		<rdfs:label>is input to</rdfs:label>
-		<skos:definition>indicates an input to some activity, process, system, report, analysis, etc.</skos:definition>
+		<skos:definition>indicates a precondition, requirement, or other contribution (e.g., data) to some activity, process, system, report, analysis, etc.</skos:definition>
```
Comment (rivettp) [inline FND/DatesAndTimes/Occurrences.rdf:260]: This is an example of why we need the new hygiene test. This whole semantic seems to be not well-formed.

File: FND/DatesAndTimes/Occurrences.rdf [hunk 4]
```diff
@@ -257,19 +257,20 @@ They are modularized this way to minimize the ontological committments that are
...
-		<skos:definition>indicates an output from some activity, process, system, report, analysis, etc.</skos:definition>
+		<skos:definition>indicates post condition or other product of some activity, process, system, report, analysis, etc.</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-dt-oc;isTriggeredBy">
 		<rdfs:label>is triggered by</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-dt-oc;OccurrenceBasedDate"/>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-oc;OccurrenceKind"/>
-		<skos:definition>An OccurrenceBasedDate is triggered by an Occurrence that exemplifies the OccurrenceKind.</skos:definition>
+		<skos:definition>is activated or initiated by</skos:definition>
```
Comment (rivettp) [inline FND/DatesAndTimes/Occurrences.rdf:272]: Again imprecise, assuming that activated and initiated have different meanings. If not, and they're synonyms, just use one of them.

**Review by rivettp — APPROVED** [review]
A few comments. Not showstoppers but if we don't address them now should be raised as issues.

**Review by trypuz — APPROVED** [review]
Approved

**Pull request merged**